cmd/meroxa/root/apps/deploy.go

```diff
@@ -71,7 +71,7 @@ type Deploy struct {
 		Path                     string `long:"path" usage:"Path to the app directory (default is local directory)"`
 		DockerHubUserName        string `long:"docker-hub-username" usage:"DockerHub username to use to build and deploy the app" hidden:"true"`         //nolint:lll
 		DockerHubAccessToken     string `long:"docker-hub-access-token" usage:"DockerHub access token to use to build and deploy the app" hidden:"true"` //nolint:lll
-		Environment              string `long:"env" usage:"environment (name or UUID) where application will be deployed to"`
+		Environment              string `long:"env" usage:"environment name where application will be deployed to"`
```

Comment:
Good callout. @anna-cross remember to run `make build` so the documentation is updated!